sys/auto_init/netif/auto_init_socket_zep.c

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (C) 2016 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ *
+ */
+
+/**
+ * @ingroup auto_init_gnrc_netif
+ * @{
+ *
+ * @file
+ * @brief   Auto initialization for @ref netdev_socket_zep devices
+ *
+ * @author  Martine Lenders <m.lenders@fu-berlin.de>
+ */
+
+#ifdef MODULE_SOCKET_ZEP
+
+#include "log.h"
+#include "socket_zep.h"
+#include "socket_zep_params.h"
+#include "net/gnrc/netif/ieee802154.h"
+
+#define ENABLE_DEBUG (0)
+#include "debug.h"
+
+/**
+ * @brief   Define stack parameters for the MAC layer thread
```

Comment:
/brief/name/ for group

Reply:
Isn't generated anyways and the other devices use the same notation (also the brace doesn't seem to be closed... I just remove the brace, because this just refers to the stack, not the prio.